creusot-contracts-dummy/src/lib.rs

```diff
@@ -33,6 +33,17 @@ pub fn snapshot(_: TS1) -> TS1 {
         .into()
 }
 
+#[proc_macro]
+pub fn ghost(_: TS1) -> TS1 {
+    // FIXME: this does not accept returning the unit type...
```

Comment:
? I don't understand what this means.

Reply:
A ghost block is supposed to return either `GhostBox<T>` or `()`. In creusot we handle it during translation to FMIR, but we also need this to be true for normal code ; but I did not find a way to do this properly. So in program code only `GhostBox<T>` can be returned...